tests/test_crm.py

```diff
@@ -138,7 +138,7 @@ def test_CrmIpv4Route(self, dvs, testlog):
         fvs = swsscommon.FieldValuePairs([("NULL","NULL")])
         intf_tbl.set("Ethernet0", fvs)
         intf_tbl.set("Ethernet0|10.0.0.0/31", fvs)
-        dvs.runcmd("ifconfig Ethernet0 up")
```

Comment:
>ifconfig [](start = 20, length = 8)

Why `ifconfig` suddenly stop working?

Reply:
it hasn't been working for a long time. a sairedis bug hide this issue.